structs/src/mapa.rs

```diff
@@ -81,6 +84,21 @@ pub enum MapaObjectVisibilityMode
     MapStationOrVisit2 = 4,
 }
 
+impl FromStr for MapaObjectVisibilityMode {
+    type Err = ();
+
+    fn from_str(input: &str) -> Result<MapaObjectVisibilityMode, Self::Err> {
```

Comment:
Please add extra mappings for the original enum values to preserve backwards compatibility. Per [Semantic Versioning](https://semver.org/), we would have to increment to randomprime v2.0 if we didn't preserve backwards compatibility here.